script.js

```diff
@@ -199,6 +199,37 @@ const scrollToAnchor = () => {
   }
 };
 
+const handleTheme = (theme) => {
+  const searchParams = new URLSearchParams(location.search);
+  searchParams.set("theme", theme);
+  history.pushState("", "theme", `?` + searchParams.toString() + location.hash);
+
+  if (theme === "dark") {
+    document.querySelector("html").classList.add("dark-mode");
+  } else if (theme === "light") {
+    document.querySelector("html").classList.remove("dark-mode");
+  }
```

Comment:
```suggestion
  if (theme === "dark") {
    document.querySelector("html").classList.add("theme--dark");
  } else if (theme === "light") {
    document.querySelector("html").classList.remove("theme--dark");
  }
```
would be better to align with excalidraw and BEM as well